Samples/Service/ZitiInstance.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenZiti;
+
+namespace OpenZiti
+{
+
+	public struct ZitiInstance
```

Comment:
This feels like it's not serving any purpose? I see it used in the Weather sample for what seems like MFA stuff... Let's move this into a "MFA sample" of its own